<launch>
 
  <node pkg="vilib_ros" type="vilib_tracker" name="vilib_tracker">
<remap from="img_in" to="cam_L/image_raw" />
  </node>

</launch>
